source/Halibut.Tests/TcpKeepAliveTests.cs

```diff
@@ -20,6 +20,7 @@ public class TcpKeepAliveTests : BaseTest
     {
         [Test]
         [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testListening:false, testWebSocket:false)]
+        // TCP Keep alives were for TcpClients, not web sockets
```

Comment:
Interesting find - side project: Do keep alives work as expected for WebSockets or did we miss something?

Reply:
Good question. I've added a [card to the board](https://app.shortcut.com/octopusdeploy/story/66528/make-keep-alives-work-with-web-sockets-if-possible) to look into this.